bin/get-config.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+try {
+  const [configEnv, path] = process.argv.slice(2)
+
+  if (!configEnv) exit('No configuration environment was passed in, make sure you supply it. Example `kaliber-config ...`')
+  if (!path) exit('No path was passed in, make sure you supply it. Example `kaliber-config ... ...`')
```

Comment:
Misschien is het duidelijker als je het deel dat wel goed is ingevuld ook toont, in voorbeeld vorm (ook al kan dat alsnog fout ingevuld zijn). Als je de waardes uit het voorbeeldje uit de readme aanhoudt:

```
  if (!configEnv) exit('No configuration environment was passed in, make sure you supply it. Example `kaliber-config ... my_config_value`')
  if (!path) exit('No path was passed in, make sure you supply it. Example `kaliber-config test ...`')
```